app-drawer/app-drawer.html

```diff
@@ -137,7 +139,8 @@
       transition: opacity 0.2s ease;
 
       -webkit-transform: translateZ(0);
-      transform:  translateZ(0);
+      transform: translateZ(0);
+      will-change: opacity;
```

Comment:
We definitely want layers here though - `contentContainer` and `scrim` _will_ be transitioned.